brink.conf

```
@@ -0,0 +1,5 @@
+BASE_REQUIREMENTS='pip==20.2.4 chevah-brink==0.78.1 paver==1.2.4'
```

We don't need chevah-brink here... only pip and paver

```suggestion
BASE_REQUIREMENTS='pip==20.2.4 paver==1.2.4'
```